README.md

```diff
@@ -163,150 +162,97 @@ déployer les services :
 En outre, il vous permet d’instancier un environnement de développement local pour ces services.
 Pour ce faire merci de prendre connaissance de la suite du document (en anglais).
 
-## Install
+## Installation
 
-### Dependencies setup
+### Datapass
 
-- [VirtualBox \^5.2.10](https://www.virtualbox.org)
-- [Vagrant \^2.1.1](https://www.vagrantup.com)
-- NFS
+#### Dependencies setup
+
+- [Docker](https://www.docker.com/)
+- [Docker compose](https://docs.docker.com/compose/install/)
 - [Ansible 2.9.10](https://www.ansible.com/)
+- [NodeJS >=16](https://nodejs.org/)
 
-### DataPass local provisioning
+#### Local environment
```

Comment:
we should point to the remote backend install here don't you think ?

Reply:
I don't really get the point of the remote backend anymore, I think the aim was to avoid having to install a whole VM with Vagrant, I think the docker requirement is now small enough to require every developper to install it?